base to pbase

akouk opened the pull request:
[@razvand](https://github.com/razvand) [@StefanJum](https://github.com/StefanJum)
 Team K.V.A.M. : [@KostasG](https://github.com/KostasG) @Foutrovic [@akouk](https://github.com/akouk) [@Monika Sara Kawka](https://github.com/monikasarakawka)

akouk committed:
base to pbase

Signed-off-by: akouk <kouka.anisa@gmail.com>

razvand reviewed (changes requested):
@akouk, please improve your commit message. Use full sentences, add a detailed description of the changes.

Also use your full name both as the commit author and for the `Signed-off-by` message.

razvand commented:
@akouk, will award points for this together with the [`app-wamr` PR](https://github.com/unikraft/app-wamr/pull/4).

akouk committed:
Update 0005-adapt-main-c-to-unikraft.patch

Changing line 79:  the source of the base address for the memory region from "base" to "pbase"

Signed-off-by: Anisa Kouka <77749065+akouk@users.noreply.github.com>

razvand commented:
Closing this as it was solved by PR #8
